applications/dashboard/views/settings/index.php

```diff
@@ -1,43 +1,27 @@
 <?php if (!defined('APPLICATION')) exit(); ?>
 <h1><?php echo t('Dashboard Home'); ?></h1>
-<?php $this->RenderAsset('Messages'); ?>
-<div class="summaries">
```

Comment:
Removing this element/class makes the previously half-width tables now full-width.  Is this intentional?